core/formatter.go

```diff
@@ -47,6 +47,17 @@ func (f *Formatter) TimeString(input time.Time) string {
 	return input.Format(f.timeLayout())
 }
 
+// PrettyDateString returns a nice representation of a given time
+// example: Mon 31. May 2021
+func (f *Formatter) PrettyDateString(input time.Time) string {
+	year := input.Local().Year()
+	day := input.Local().Day()
+	weekday := input.Local().Weekday()
+	month := input.Local().Month()
+
+	return fmt.Sprintf("%v %d. %v %d", weekday, day, month, year)
```

Comment:
You can replace this with

```suggestion
	return input.Format("Monday, 02. January 2006")
```